cpp/tests/utilities/column_utilities.cu

```diff
@@ -176,21 +173,19 @@ void print_differences(thrust::device_vector<int> const& differences,
              << differences[i] << "] = " << h_right_strings[i] << std::endl;
     }
 
-    EXPECT_EQ(differences.size(), size_t{0}) << buffer.str();
+    return buffer.str();
   } else {
-    //  If there are differences, just display the first one
-    int index = differences[0];
+    int index = differences[0];  // only stringify first difference
 
     auto diff_lhs = cudf::detail::slice(lhs, index, index + 1);
     auto diff_rhs = cudf::detail::slice(rhs, index, index + 1);
 
     std::vector<std::string> h_left_strings  = to_strings(diff_lhs);
     std::vector<std::string> h_right_strings = to_strings(diff_rhs);
```

Comment:
These 2 vectors are not used in this block.

Reply:
Compiler might be issuing a warning for this.

Reply:
Great catch. Removed :)